openwisp_utils/admin_theme/templates/admin/base_site.html

```diff
@@ -45,6 +45,21 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header }}</a></h1>
     <a href="{{ item.url }}"
        class="{{ item.class }}">{{ item.label }}</a>
     {% endfor %}
+    {% for group in openwisp_menu_groups %}
+        <div class="group-name">{% if group.icon %}
+            <img src={% static group.icon %} alt="icon" />
```

Comment:
I took for granted that we'd implement icons with CSS since CSS is more powerful and can give us more control, so the icon attribute should just be a class which is used to specify which icon has to be used.